external-crates/move/crates/move-compiler/src/shared/ide.rs

```diff
@@ -47,13 +47,24 @@ pub struct MacroCallInfo {
     pub by_value_args: Vec<T::SequenceItem>,
 }
 
+#[derive(Debug, Clone, PartialOrd, Ord, PartialEq, Eq)]
+pub struct AutocompleteMethod {
+    pub method_name: Symbol,
+    pub target_function: (E::ModuleIdent, P::FunctionName),
+}
+
+#[derive(Debug, Clone)]
+pub struct AutocompleteFields {
+    pub enclosing_type: N::Type,
+    pub names: BTreeSet<Symbol>,
+}
```

Comment:
It looks like this isn't being used at the moment.

Reply:
Indeed! Good catch!